python/pynessie/cli.py

```diff
@@ -46,10 +46,22 @@
 class ContextObject(object):
     """Click context object."""
 
-    nessie: NessieClient
+    config: confuse.Configuration
     verbose: bool
     json: bool
 
+    __nessie: Optional[NessieClient] = None
+
+    def nessie(self: "ContextObject") -> NessieClient:
+        """Get the ``NessieClient`` instance, create it, if necessary."""
+        if not self.__nessie:
```

Comment:
IIRC, double-underscore is "private" and single-underscore is "protected"

Reply:
it means that the name will be mangled by the interpreter, not that its private. `__` is used to avoid name clashes w/ subclasses. `_` is used for non-public members. We prefer `_` over `__` unless the variable really shouldn't be used from subclasses. https://www.python.org/dev/peps/pep-0008/#id47

Reply:
Updated to `_`